src/commands/create.ts

```diff
@@ -26,29 +31,43 @@ const command: GluegunCommand = {
   name: 'create',
   alias: ['c', 'generate', 'g'],
   run: async (toolbox: GluegunToolboxExtended) => {
-    const { parameters, prompt, print } = toolbox;
+    const { filesystem, parameters, prompt, print } = toolbox;
     const { loadWhile } = interfaceHelpers(toolbox);
     const { checkCurrentDirReactNativeProject } = validations(toolbox);
 
     await loadWhile(checkCurrentDirReactNativeProject());
 
-    const userCategoryInput = parameters.first;
+    const userCategoryInput = parameters.first.toLowerCase();
```

Comment:
If `parameters.first` does not exist, this will throw an err. If the first parameter is always required, we may want to assert/validate parameters before this to make the error message more clear.

Reply:
good catch, I'll fix this

Reply:
fixed ✅